### PR TITLE
#837 fixed creator crash

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: evaluator/abstract_evaluator.py [hunk 1]
```diff
@@ -66,6 +66,12 @@ def get_config_file_name(cls, config_evaluator_type=None):
         return f"{TENTACLES_PATH}/{TENTACLES_EVALUATOR_PATH}/{config_evaluator_type}/{EVALUATOR_CONFIG_FOLDER}" \
             f"/{cls.get_name() + CONFIG_FILE_EXT}"
 
+    @classmethod
+    def get_config_file_error_message(cls, error):
```
Comment (Herklos) [inline evaluator/abstract_evaluator.py]: Where this method is used ?

Comment (Herklos) [reply evaluator/abstract_evaluator.py]: Oh ok in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/60

Comment (Herklos) [reply evaluator/abstract_evaluator.py]: Why this is not used at all evaluator config load ?

Comment (GuillaumeDSM) [reply evaluator/abstract_evaluator.py]: Because no one coded it in evaluator config load yet :)

**Pull request merged**